### DISCUSSION TITLE
[Bug] Embedded Images metadata not showing...

### DISCUSSION THREAD

**Discussion by DanielNoamTuby** [pimcore/pimcore#15893] [Q&A / Support]
1) I have images with metadata but when I upload I'm not seeing that.
2) If I add custom metadata and download the image I dont see the new metadata.

please help me solve this...

**Comment by kingjia90** [comment]
Not sure, sounds like mixing up `Meta Info` with Pimcore `Custom MetaData`?
The embedded meta info AFAIK cannot be edited on Pimcore backend, is read-only, there may be external tool to do that and the info in there are provided by the tool who generate the image, some compression tool may remove these to reduce file size.
![image](https://github.com/pimcore/pimcore/assets/6014195/fabda692-3252-4917-9a7c-aefb07f3250c)

The Custom MetaDatas are not "integrated" into a file when it is downloaded/uploaded (that would potentially behave like a malware). 
It's only a database entry associated to the asset id.
![image](https://github.com/pimcore/pimcore/assets/6014195/04aca546-5a28-4646-ab30-dbe8d2b6edcc)
